index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+        
```

Comment:
It's a great attempt but there some minor changes needed.
-Here there is a space not needed. Please kindly remove it.

Reply:
- [ ] Good job on making the suggestion to remove the empty space, however, it would be helpful to provide a reason for the suggested change. The reason could be something like "The extra space affects the formatting consistency of the code." 🎯